docs/manual/scala/guide/cluster/code/ChildActors.scala

```diff
@@ -0,0 +1,124 @@
+package docs.scaladsl.cluster.childactors
+
+import akka.actor.{Actor, ActorSystem, Props}
+import akka.cluster.sharding.{ClusterSharding, ClusterShardingSettings, ShardRegion}
+import akka.testkit.{TestKit, TestProbe}
+import akka.util.Timeout
+import com.lightbend.lagom.scaladsl.persistence.PersistentEntity.ReplyType
+import com.lightbend.lagom.scaladsl.persistence.{ChildPersistentEntity, ChildPersistentEntityFactory, PersistentEntity}
+import org.scalatest.{BeforeAndAfterAll, WordSpecLike}
+
+//#create-child-persistent-entity-factory
+class MyProcessManagerRouter(system: ActorSystem, entityFactory: () => MyProcessManagerEntity) {
```

Comment:
The current CSS cuts ` entityFactory: () => MyProcessManagerEntity` and it seems the code actually reads ` entityFactory: () => MyProcessManager` which is confusing. Could we rename the classes or move this argument to a new line?